pom.xml

```diff
@@ -57,6 +57,7 @@
     <dependency>
       <groupId>org.opengis.cite.teamengine</groupId>
       <artifactId>teamengine-spi</artifactId>
+      <version>${docker.teamengine.version}</version>
```

Comment:
@ajanett Can you revert this change? The docker.teamengine.version should be used only by the docker plugin.